### PR TITLE
 Improved the track Screen

### PR DESCRIPTION
**Pull request by Albinseban2021**
Improved the track screen in mobile view

### CODE REVIEW

File: lib/Widgets/truckHistoryStatus.dart [hunk 1]
```diff
@@ -99,8 +92,8 @@ class _TruckStatusState extends State<TruckStatus> {
                           crossAxisAlignment: CrossAxisAlignment.start,
                           children: [
                             Container(
-                                width:
-                                    MediaQuery.of(context).size.width / 4 - 74,
+                                width: 300,
+                                // MediaQuery.of(context).size.width / 4 - 74,
```
Comment (Anik-Chakraborty) [inline lib/Widgets/truckHistoryStatus.dart:96]: Remove these commented out code if they are not required

File: lib/Widgets/truckHistoryStatus.dart [hunk 2]
```diff
@@ -226,8 +219,8 @@ class _TruckStatusState extends State<TruckStatus> {
...
```
Comment (Anik-Chakraborty) [inline lib/Widgets/truckHistoryStatus.dart:223]: Same as here and other places

**Comment by Anik-Chakraborty** [comment]
Found this error or terminal... resolve this
Error: DioException [connection error]: The connection errored: The XMLHttpRequest onError callback was called. This typically indicates an error on the network layer.
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 288:49  throw_

**Comment by Anik-Chakraborty** [comment]
Still below error is there and download button not working
Error: DioException [connection error]: The connection errored: The XMLHttpRequest onError callback was called. This typically indicates an error on the network layer.
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 288:49  throw_

Cross check your spelling while creating commit messages😶

**Comment by Anik-Chakraborty** [comment]
Got this error in web ---> screen size : mobile
image

**Comment by Albinseban2021** [comment]
issue resolved

**Review by Anik-Chakraborty — APPROVED** [review]

**Review by liveasy97 — APPROVED** [review]

**Pull request merged**